wunderline-add.js

```diff
@@ -92,6 +92,10 @@ function main() {
   if (app.due) {
     if (moment(app.due, "YYYY-MM-DD", true).isValid()) {
       dueDate = moment(app.due);
+    } else if (moment(app.due, "MM-DD", true).isValid()) {
+      var date = new Date();
+      var year = date.getFullYear();
+      dueDate = year + moment(app.due);
```

Comment:
It doesn't look like this is working like you expect it to.

Reply:
How do I test on my machine? I never worked with javascript before 😅

Reply:
Clone your fork, run `npm install` and use `node wunderline.js` where you would normally use just `wunderline`.

Reply:
Thanks!